### ISSUE TITLE
JavaDoc for Class Properties

### ISSUE THREAD

**Issue by domschmidt** [vojtechhabarta/typescript-generator#400]
Hey there,

thank you very much for this library. I used it already very often and it seems to be very stable.

at least parts of the javadoc would be very helpful on class properties or enums :-)

situation:

javadoc.xml
```
<package name="....datatable.enums">
        <enum name="EnumDataType" qualified="....datatable.enums.EnumDataType" scope="public" included="true">
            <comment>Definiert den Datentyp für die Zellenspalte.</comment>
            <class qualified="java.lang.Enum">
                <generic qualified="....datatable.enums.EnumDataType"/>
            </class>
            <constant name="TEXT"/>
            <constant name="NUMBER"/>
            <constant name="EMAIL">
                <comment>Email Format xx..</comment>
            </constant>
            <constant name="ISODATE">
                <comment>siehe ISO-Format</comment>
            </constant>
        </enum>
```

expected result:
```

    /**
     * Definiert den Datentyp für die Zellenspalte.
     */
    type EnumDataType = 
        "TEXT" 
        | "NUMBER" 
        | "EMAIL"  /* Email Format xx.. */
        | "ISODATE"; /* siehe ISO-Format */
```

actual result:
```

    /**
     * Definiert den Datentyp für die Zellenspalte.
     */
    type EnumDataType = "TEXT" | "NUMBER" | "EMAIL" | "ISODATE";

```
Thank you for your great work.

Best,
Dominik

**Comment by vojtechhabarta** [comment]
Hey Dominik,

Javadoc on properties is already supported. Javadoc comment should be on member which defines the property. This is similar to issue with `@ApiModelProperty` annotation, see https://github.com/vojtechhabarta/typescript-generator/issues/386#issuecomment-514980492. Also see [JavadocTest.java](https://github.com/vojtechhabarta/typescript-generator/blob/master/typescript-generator-core/src/test/java/cz/habarta/typescript/generator/JavadocTest.java) for examples of Javadoc on properties.

Javadoc on enum constants (string literals in union type) is not generated because it is not used by editors like VS Code. So it is not clear if it is better to generate it as you are suggesting or if it should be somehow added to Javadoc on enum type for example like this:
``` typescript
/**
 * Definiert den Datentyp für die Zellenspalte.
 * 
 * Values:
 * - `"TEXT"`
 * - `"NUMBER"`
 * - `"EMAIL"` - Email Format xx..
 * - `"ISODATE"` - siehe ISO-Format
 */
type EnumDataType = "TEXT" | "NUMBER" | "EMAIL" | "ISODATE";
```

**Comment by domschmidt** [comment]
I'd love your idea and proposal. That would be great!

**Comment by vojtechhabarta** [comment]
Released in version [2.17.558](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.17.558).

**Comment by domschmidt** [comment]
ohh perfect! thank you
